src/main/kotlin/Menu.kt

```diff
@@ -0,0 +1,51 @@
+class Menu<T>(private val title: String, private val items: MutableList<T>, private val onItemSelected: (T) -> Unit, private val onCreateNewItem: () -> T) {
```

Comment:
Если в классе больше 1 параметра - они пишутся с новой строки
```suggestion
class Menu<T>(
    private val title: String, 
    private val items: MutableList<T>, 
    private val onItemSelected: (T) -> Unit, 
    private val onCreateNewItem: () -> T
) {
```